security/p11-kit/Portfile

```diff
@@ -29,15 +27,8 @@ depends_lib         port:desktop-file-utils \
 
 depends_run         path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
 
-patchfiles          patch-p11-kit-Makefile.am.diff \
-                    patch-common-compat.c.diff
-
-use_autoreconf      yes
-autoreconf.args     -fvi
+patchfiles          patch-common-compat.c.diff
 
 configure.args      --with-trust-paths=${prefix}/share/curl/curl-ca-bundle.crt:${prefix}/etc/openssl \
                     --disable-silent-rules
 
```

Comment:
Please remove this empty line.

Reply:
Not to make a storm in a teacup, but I don't follow. The way I see it, as long as `port lint` found no issues, there's no written guidelines saying otherwise, it boils down to a stylistic personal preference of use of whitespace — since I'm the maintainer for the port, it seems that my preference on the matter should be as good as any. Is there something that I'm missing here?

Reply:
Never mind then. This Portfile didn't have a trailing empty line before your changes so I thought you overlooked it, plus that most Portfiles in this ports tree follow this convention.

Reply:
That's all right. If you don't mind me making a suggestion (one of the things I do as part of my daily job is reviewing other's code): it helps the receiving side a lot if you state *why* you are requesting/suggesting a particular change — otherwise it might be perceived as being arbitrary and/or based on personal preference, even if it isn't.

In this particular case I would have said "This empty line here, is it deliberate or an oversight? Most ports in the tree keep these two keywords together". I'm not sure if you agree, but in my experience it makes a big difference in terms of communication.

Regarding the diff, I look over it twice now and I didn't actually remove or add any empty lines — all the whitespace is exactly as it was before. However, if there is indeed a convention being followed, even though it is not written on the guidelines, I want to follow it as well so I have no problem with amending the PR. Right now the keywords are in separate "sections" because they control different phases.

Reply:
I fully agree. However I'm reviewing this PR during 5-minute breaks under horrible network condition.

Regarding your changes, the new Portfile ends with two `\n`s:
```
00000530: 2d73 696c 656e 742d 7275 6c65 730a 0a    -silent-rules..
```
which produces an empty line at end of file.

Reply:
Note that the comment is made on line 34 of the new Portfile, not line 31.

GitHub doesn't show any context after the line being commented.

Reply:
That just added to my confusion since I got the impression that you were referring to a different line. I have amended the PR.